fetch_contigs.py

```diff
@@ -0,0 +1,35 @@
+import sys
+import os
+import urllib.request
+from Bio import Entrez
+
+def main(email=None, api_key=None, retmax=None):
+    Entrez.email = email
+    Entrez.api_key = api_key
+
+    handle = Entrez.esearch(db="assembly", term = "E. coli[Organism] AND contig[Assembly Level]", retmax = num)

```

Comment:
E. coli is the organism of contigs that we want to download, this should be a command line parameter so that we could download other organisms contigs if we wanted for example Salmonella Enterica

Reply:
If "E. coli" is encoded as a parameter, how do we warn users of mistakes due to spelling, case sensitivity, indentation as such?

Reply:
You could add a method for validating the parameter. For example a simple regex expression that allows certain characters then just catch error from efetch

Reply:
resolved this by modifying 
term = genus+" "+species+"[Organism] AND contig[Assembly Level]",

Upon entering right combination of genus and species (E. coli or Escherichia coli) in command line argument, the program returns without error. Otherwise captures empty list from esearch and returns 
"Incorrect organism name. Check for spelling, case sensitivities and indentations."